modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18472,6 +18472,7 @@ this-document-type-does-not-belong-to-this-site.-you-may-affect-other-sites-if-y
 this-editor-is-not-optimized-for-mobile-or-400-zooming=This editor is not optimized for mobile or 400% zooming.
 this-event-and-any-future-events-in-the-series-will-be-modified=This event, and any future events in the series, will be modified.
 this-event-does-not-repeat=This event does not repeat.
+this-external-reference-code-is-already-in-use=This External Reference Code is already in use. Try another one.
```

Comment:
Could you change this language key to `this-external-reference-code-is-already-in-use-try-another-one`, please?

This key is follow the same pattern [here](https://github.com/liferay/liferay-portal/blob/cb1a32e11e9579147c26ddcca5df579f6d14fc5d/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties#L18541) and [here](https://github.com/liferay/liferay-portal/blob/cb1a32e11e9579147c26ddcca5df579f6d14fc5d/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties#L11107).